app/templates/index.html

```diff
@@ -41,5 +41,48 @@
         </div>
         <h1>{{ title }}</h1>
     </div>
+
+    <div>
+        <style>
+            h2 {
+                margin-right: 10px;
+                margin-left: 15px;
+                color: #1C539F;
+                font-size: 40px;
+                text-align: center;
+                text-underline-offset: 5px;
+            }
+            h3 {
+                font-size: 25px;
+            }
+        </style>
+        <h2><u>About Us</u></h2>
+        <h3>Aerin Brown</h3>
+        <p>Aerin Brown is an incoming second-year Software Engineering student at McGill University, and attends as a 
```

Comment:
Just a thought here, rather than writing all this text here in the html file wouldn't it be better to use a text file and use that text using

```
<object data="file.txt" />
```

Although the text styling might not work, what do you folks think about this?

Reply:
It would be a nice implementation to make everything templated and easy to edit. Awesome idea! 💡

Reply:
I tried to do that initially, but as Prad suspected, the text styling doesn't work at all and it looks very jarring. It ends up looking something like this:
<img width="945" alt="About Us Text File Problem" src="https://github.com/MAMV3x3/MLH-Fellowship-pe-portfolio-site/assets/72281505/45010b14-4aab-466f-9b41-787c749b94da">

I've done some research into ways to get around this, but they all seem to require using php, and I have no idea how one would integrate it into a python-based website 😅. If any of you know a way to fix this, I'd be happy to implement it!

Reply:
Maybe you could either use a directory or an external file for each one to put its description text and via a Jira template generate an object for each one, so you can leave the content wrapped in h3 and p and use templates for information. 👀

Reply:
Ohhh I should've thought of that haha I'll get right on that! Thanks!